pkg/middleware/is_duplicated_middleware.go

```diff
@@ -0,0 +1,29 @@
+package middleware
+
+import (
+	"fmt"
+	"github.com/gofiber/fiber/v2"
+	restError "github.com/kotalco/community-api/pkg/errors"
+	"github.com/kotalco/community-api/pkg/k8s/statefulset"
+)
+
+var statefulService = statefulset.NewService()
+
+func IsDuplicated(c *fiber.Ctx) error {
+	if c.Request().Header.IsPost() { // check on posts verbs only while trying to create new resource
+		var bodyFields map[string]interface{}
+		_ = c.BodyParser(&bodyFields)
```

Comment:
If you're interested in the error, no need for the assignment.
```go
c.BodyParser(&bodyFields)
```

Reply:
i am using the blank identifier , to show the developers i am explicitly ignoring the error.